production-dependency-templates/exocom/README.md

```diff
@@ -0,0 +1,53 @@
+# Exocom dependency
+
+_Production plugin for RDS-type dependencies_
+
+## Usage
+- List database `exocom` as a remote dependency in `application.yml`, with `type: exocom`
+  - Populate required `template-config` fields (defined below)
+- List any exocom-specific service data under `dependency-data.exocom` in a service's `service.yml`
+
+### Required fields
+The following fields should be populated by the user.
+
+#### Template configuration in application:
+- `version`: Define which version of Exocom to use
+Example:
+```
```

Comment:
please add syntax highlighting to make this easier to read. Same for other blocks